### PR TITLE
Support selecting versions in /api/bulktext. 

### PR DESCRIPTION
**Pull request by ronshapiro**
Fixes https://github.com/Sefaria/Sefaria-Project/issues/1387

### CODE REVIEW

File: sefaria/views.py [hunk 1]
```diff
@@ -379,8 +381,13 @@ def bundle_many_texts(refs, useTextFamily=False, as_sized_string=False, min_char
                     'url': oref.url()
                 }
             else:
-                he_tc = model.TextChunk(oref, "he", actual_lang=translation_language_preference)
-                en_tc = model.TextChunk(oref, "en", actual_lang=translation_language_preference)
+                he_tc = model.TextChunk(oref, "he", actual_lang=translation_language_preference, vtitle=hebrew_version)
+                en_tc = model.TextChunk(oref, "en", actual_lang=translation_language_preference, vtitle=english_version)
+                if hebrew_version and he_tc.is_empty():
+                  raise ValueError(f"{oref.normal()} does not have the Hebrew version: {hebrew_version}")
```
Comment (ronshapiro) [inline sefaria/views.py:387]: Is there a better error type to raise here so that the user sees the error?

Comment (ronshapiro) [reply sefaria/views.py]: Done :)

**Comment by edamboritz** [comment]
I'm inclined to approve this because it seems to be a small enough change that will bring benefit to API users. 

That being said, the bulktext api endpoint suffers from the same problems as the current main texts api endpoint, which is that it makes some outdated assumptions about versions. 

It was also not designed primarily to be used too often, outside of the few places inside Sefaria it's currently used, as caching its responses is difficult by nature.

**Review by edamboritz — APPROVED** [review]

**Comment by ronshapiro** [comment]
I know not to expect much from the bulktext API. This felt like a conveniently small change.

I'm indifferent to caching - the requests I make have no need to be cached. They should be quite unique.

**Comment by ronshapiro** [comment]
Thanks! How often are releases so I know when it's in prod?

בתאריך יום ה׳, 1 ביוני 2023, 12:11, מאת Ephraim Damboritz ‏<
***@***.***>:

**Pull request merged**